yaptide/routes.py

```diff
@@ -37,20 +38,28 @@ class ShieldhitDemo(Resource):
     """Class responsible for Shieldhit Demo running"""
 
     @staticmethod
-    def get():
+    def post():
         """Method handling running shieldhit with server"""
         shschema = SHSchema()
         args = request.args
         errors = shschema.validate(args)
         if errors:
             return errors
-
         param_dict = shschema.load(args)
-        simulation_result = run_shieldhit(param_dict)
+
+        json_data = request.json
+        if not json_data:
+            return ({"message": "Input Json Error"},
```

Comment:
Why are you returning tuples? Also, you can just return the codes as numbers.

Reply:
If I send some request I would like to know what went wrong. Tuple is used to provide info about it. Talking about returning status codes it is just a way of doing it.